### PR TITLE
Invoice getversion + remove inv protobufs

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: <Github link>

### CODE REVIEW

File: documents/invoice/test_inv.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
+// +build integration unit testworld
```
[the remark this answers is not in the document]
Comment (vedhavyas) [reply documents/invoice/test_inv.go]: Lets also add this file here - https://github.com/centrifuge/go-centrifuge/blob/develop/codecov.yml#L7 ?

Helps us avoid unnneded code coverage issues

File: documents/invoice/bootstrapper_test.go [hunk 2]
```diff
@@ -4,27 +4,11 @@ package invoice
...
-	//registry := document.GetRegistryInstance()
-	//
-	//documentType, err := cd.GetTypeUrl(coreDocument)
-	//assert.Nil(t, err, "should not throw an error because document contains a type")
-	//
-	//service, err := registry.LocateService(documentType)
-	//assert.Nil(t, err, "service should be successful registered and able to locate")
-	//assert.NotNil(t, service, "service should be returned")
+	err := (&Bootstrapper{}).Bootstrap(map[string]interface{}{})
+	assert.Error(t, err, "Should throw an error because of empty context")
```
Comment (vedhavyas) [inline documents/invoice/bootstrapper_test.go]: Can we have better tests for this bootstapper?

File: documents/invoice/test_inv.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	}
+}
+
+func InitInvoice(t *testing.T, did identity.DID, payload documents.CreatePayload) *Invoice {
+	inv := new(Invoice)
+	assert.NoError(t, inv.unpackFromCreatePayload(did, payload))
+	return inv
+}
+
+func CreateInvWithEmbedCDWithPayload(t *testing.T, ctx context.Context, did identity.DID, payload documents.CreatePayload) (*Invoice, coredocumentpb.CoreDocument) {
```
Comment (vedhavyas) [inline documents/invoice/test_inv.go]: we seem to have different names for invoice in functions - `Inv` and `Invoice`. Let's be consistent?

Comment (vedhavyas) [reply documents/invoice/test_inv.go]: I would recommend `Invoice`

File: documents/invoice/bootstrapper_test.go (hunk 2 above)
Comment (vimukthi-git) [reply documents/invoice/bootstrapper_test.go]: these are just copied over stuff, besides testing is just mocking stuff on the context which is just pointless.

**Pull request merged**